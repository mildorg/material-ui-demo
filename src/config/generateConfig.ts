export function generateConfig() {
  return {
    server: "http://localhost",
    port: 8080,
    domain: "localhost",
    timer: "20191",
  };
}

export function generateAnotherConfig() {
  return {
    server: "http://localhost",
    port: 8080,
    domain: "localhost",
    timer: new Date(),
  };
}
