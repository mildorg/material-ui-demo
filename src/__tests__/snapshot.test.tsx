import { generateConfig, generateAnotherConfig } from "../config";

describe("jest 快照测试", () => {
  test("测试 generateConfig", () => {
    expect(generateConfig()).toMatchSnapshot();
  });

  test("测试generateAnotherConfig", () => {
    expect(generateAnotherConfig()).toMatchSnapshot({
      timer: expect.any(Date),
    });
  });
});
