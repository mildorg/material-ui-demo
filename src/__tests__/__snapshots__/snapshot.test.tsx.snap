// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jest 快照测试 测试 generateConfig 1`] = `
Object {
  "domain": "localhost",
  "port": 8080,
  "server": "http://localhost",
  "timer": "20191",
}
`;

exports[`jest 快照测试 测试generateAnotherConfig 1`] = `
Object {
  "domain": "localhost",
  "port": 8080,
  "server": "http://localhost",
  "timer": Any<Date>,
}
`;
